tesseract/tesseract_planning/tesseract_process_managers/src/process_managers/simple_process_manager.cpp

```diff
@@ -0,0 +1,97 @@
+
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <functional>
+#include <taskflow/taskflow.hpp>
+#include <fstream>
+TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
+#include <tesseract_process_managers/process_managers/simple_process_manager.h>
+
+#include <tesseract_process_managers/process_generators/motion_planner_process_generator.h>
+#include <tesseract_process_managers/taskflow_generators/sequential_taskflow.h>
+
+#include <tesseract_motion_planners/trajopt/profile/trajopt_default_plan_profile.h>
+#include <tesseract_motion_planners/trajopt/profile/trajopt_default_composite_profile.h>
+#include <tesseract_motion_planners/trajopt/problem_generators/default_problem_generator.h>
+#include <tesseract_process_managers/process_managers/default_processes/default_freespace_processes.h>
+
+using namespace tesseract_planning;
+
+SimpleProcessManager::SimpleProcessManager(TaskflowGenerator::UPtr taskflow_generator, std::size_t n)
+  : taskflow_generator_(std::move(taskflow_generator)), executor_(n), taskflow_("SimpleProcessManagerTaskflow")
+{
+}
+
+bool SimpleProcessManager::init(ProcessInput input)
+{
+  // Clear the process manager
+  clear();
+
+  // Check the overall input
+  if (!isCompositeInstruction(*(input.instruction)))
+  {
+    CONSOLE_BRIDGE_logError("ProcessInput Invalid: input.instructions should be a composite");
+    return false;
+  }
+  const auto* composite = input.instruction->cast_const<CompositeInstruction>();
+
+  // Check that it has a start instruction
+  if (!composite->hasStartInstruction() && input.start_instruction_ptr == nullptr &&
+      isNullInstruction(input.start_instruction))
+  {
+    CONSOLE_BRIDGE_logError("ProcessInput Invalid: input.instructions should have a start instruction");
+    return false;
+  }
+
+  // Create the dependency graph
+  input.instruction->print("Generating Taskflow for: ");
+  auto task = taskflow_
+                  .composed_of(taskflow_generator_->generateTaskflow(
+                      input, [this]() { successCallback(); }, [this]() { failureCallback(); }))
+                  .name("freespace");
```

Comment:
Perhaps default name should be "simple"?

Reply:
It has been renamed.